src/components/ArticleLink.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { foreground, pink } from '@utils/theme';
+
+import { Link } from './Link';
+
+const StyledLink = styled(Link)`
+  border-left: 3px solid ${pink};
+  display: block;
+  margin: 2rem 0;
+  padding: 2rem;
+
+  :hover {
+    text-decoration: none;
+  }
+
+  p {
+    color: ${foreground};
+
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+`;
+
+export function ArticleLink({ childMarkdownRemark, name }) {
+  const { title, description } = childMarkdownRemark.frontmatter;
+
+  return (
+    <StyledLink to={`/blog/${name}`}>
+      <article>
+        <h2>{title}</h2>
+        {description ? <p>{description}</p> : null}
+      </article>
```

Comment:
I don't think `article` is appropriate as this doesn't contain the entire content of the article.
```suggestion
      <h2>{title}</h2>
      {description ? <p>{description}</p> : null}
```

Reply:
The HTML spec says that `article` can either be `complete` or `self-contained`; it's definitely a best practice to wrap each post in a list like this in an `article` tag IMO!